Source/WebCore/page/FocusController.h

```diff
@@ -50,6 +50,12 @@ class TreeScope;
 
 struct FocusCandidate;
 
+enum class ContinuedSearchInRemoteFrame : bool { No, Yes };
+struct FindFocusableElementResult {
```

Comment:
What's the intended values for when the next focusable element in another process?
Are we gonna return nullptr and `ContinuedSearchInRemoteFrame::Yes`?
If so, maybe `ContinuedSearchInRemoteFrame` should be renamed to `ContinueSearchInRemoteFrame`.
It's also inconsistent that this is called "find" focusable element result whereas the enum above is called continued "search" in remote frame.
Maybe rename this to `FocusableElementSearchResult`?

Reply:
"ContinuED" <-- as in, past tense -- actually will be accurate, as when the condition is detected is when we'll call out to the remote frame.

(That code has been written, I'm just trying to land in smaller, more manageable patches)

FocusableElementSearchResult is a decent naming suggestion! Will update as I also fight EWS.